pkg/controller/common/util.go

```diff
@@ -452,6 +452,19 @@ func GetVolumeMode(pvc *corev1.PersistentVolumeClaim) corev1.PersistentVolumeMod
 	return util.ResolveVolumeMode(pvc.Spec.VolumeMode)
 }
 
+// GetStorageClass returns the StorageClassName from DataVolume PVC or Storage spec
+func GetStorageClass(dv *cdiv1.DataVolume) *string {
```

Comment:
GetStorageClass is a bit general, maybe GetStorageClassFromDVSpec

Reply:
renamed